### PR TITLE
(feat) Added grunt support for building locally and production

### LINKED ISSUE #9: Set up Grunt file

### PR DESCRIPTION
**Pull request by rmagee88**
Fix #9, #142 
Lots going on here, but it works and makes the build process much simpler!

grunt commands we can now use:
grunt devSocket (if you want to run with sockets)
grunt devCc (if you want to run with chromecast)
--api=prod (this flag will use our prod API to get/create memes instead of your local server)

grunt prod (only for the heroku when we deploy)

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: node index.js
+web: grunt prod
```
Comment (ajkitson) [inline Procfile:1]: Does Heroku have grunt installed or do we need to add it to package.json?

Comment (ajkitson) [inline Procfile:1]: Nevermind--I see it's in devDependencies below :cactus:

**Comment by ajkitson** [comment]
LGTM. :pig:

File: Procfile (hunk 1 above)
Comment (i-made-that) [inline Procfile:1]: :walking:

**Pull request merged**